webserver/views/api/v1/core.py

```diff
@@ -6,21 +6,21 @@
 from webserver.decorators import crossdomain
 import webserver.exceptions
 import json
-import uuid
 
-api_bp = Blueprint('api', __name__)
+bp_core = Blueprint('api_v1_core', __name__)
 
 
-@api_bp.route("/<uuid:mbid>/count", methods=["GET"])
+@bp_core.route("/<uuid:mbid>/count", methods=["GET"])
 @crossdomain()
 def count(mbid):
+    """Endpoint for getting number of submissions for a given recording."""
     return jsonify({
         'mbid': mbid,
-        'count': count_lowlevel(mbid),
+        'count': count_lowlevel(str(mbid)),
     })
 
 
-@api_bp.route("/<string:mbid>/low-level", methods=["GET"])
+@bp_core.route("/<uuid:mbid>/low-level", methods=["GET"])
```

Comment:
Good point. We shouldn't remove this page completely (because it has other things like the highlevel model descriptions), but we can link specific API stuff to the RTD docs. Should we also move the example files there? Or perhaps we can just link to a json view of an existing document anyway.